src/store/modules/order/actions.ts

```diff
@@ -218,6 +222,11 @@ const actions: ActionTree<OrderState, RootState> = {
       payload.json.filter = payload.json.filter.concat(` AND autoCancelDate: [${state.currentOrderFiltersSelected.autoCancelDate + 'T00:00:00Z'} TO ${state.currentOrderFiltersSelected.autoCancelDate + 'T23:59:59Z'}]`)
     }
 
+    const correspondingPoId = state.currentOrderFiltersSelected.poIds.map((id: string) => state.availableOrderFilterOptions.poIds[id]).toString().replaceAll(",", " OR ")
```

Comment:
We could use join() method with ' OR ' as separator
https://stackoverflow.com/questions/39771131/how-to-convert-array-into-comma-separated-string-in-javascript

Reply:
Replaces the usage of replaceAll with join